documentation/test-scenario/references/TestDefinitionFileSchema.json

```diff
@@ -0,0 +1,363 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/definitions/VariableScope"
+    }
+  ],
+  "properties": {
+    "scope": {
+      "type": "string",
+      "enum": [
+        "ResourceGroup"
+      ]
+    },
+    "requiredVariables": {
```

Comment:
I see 'requiredVariables' is defined in both 'testDefinitionFile' and 'TestScenario'.
What's the relationship of both of them? Can I expect TestScenario.requiredVariables should always be a subset of testDefinitionFile.requiredVariables ?

Reply:
It should be a superset of file requiredVariables. Please use testScenario.requiredVariables.